config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
   resources :sites do
-    resources :reviews
+    resources :reviews, only: [:index, :new, :create, :edit, :update, :destroy]
```

Comment:
excellent configuration of your routes. this is really the ideal and most adequate organization of concerns